### PR TITLE
Finestra Miner branding requirements added.

### PR DESCRIPTION
**Pull request by encku**

### CODE REVIEW

**Comment by tyler-whitman** [comment]
Once you have a maker account you will also need to add your support email here https://github.com/helium/hotspot-app/blob/main/src/utils/stakingClient.ts#L55

**Review by tyler-whitman — APPROVED** [review]
Code looks good and is working as expected, just missing the support email once you have a maker ID. 

I will wait for one other approver as well.

**Comment by cokes518** [comment]
FYI, we'll merge this to main when community has approved Smart Mimic as a Maker.

**Review by tyler-whitman — CHANGES_REQUESTED** [review]
Once you have a maker account you will also need to add your support email here https://github.com/helium/hotspot-app/blob/main/src/utils/stakingClient.ts#L55

**Comment by cokes518** [comment]
As part of assert_v2, you will also want to add your Hotspot and its default antenna dBi:
https://github.com/helium/hotspot-app/blob/da5fbce2608fb3a23347ffbde556d2829dce9f29/src/components/HotspotConfigurationPicker.tsx#L26

and the EN string
https://github.com/helium/hotspot-app/blob/da5fbce2608fb3a23347ffbde556d2829dce9f29/src/locales/en.ts#L959

**Review by tyler-whitman — CHANGES_REQUESTED** [review]
Thanks for adding the support email. The last change would be the antenna info with Assert V2 that Coco mentioned

**Comment by tyler-whitman** [comment]
You can reference https://github.com/helium/hotspot-app/pull/497 for the antenna changes

**Comment by encku** [comment]
> You can reference #497 for the antenna changes



> Thanks for adding the support email. The last change would be the antenna info with Assert V2 that Coco mentioned. Since this is a new change. I can also do it for you if you give me the values you would like me to add.

#497 completed.

**Comment by cokes518** [comment]
@tyler-whitman this can be merged

File: src/components/HotspotConfigurationPicker.tsx [hunk 1]
```diff
@@ -15,6 +15,37 @@ import { decimalSeparator, groupSeparator, locale } from '../utils/i18n'
 import { useColors } from '../theme/themeHooks'
 import { Antenna, Antennas } from '../constants/antennas'
 
+export type AntennaId =
```
Comment (tyler-whitman) [inline src/components/HotspotConfigurationPicker.tsx:18]: You need to remove these lines, 18 - 48.

They got moved to the `constants/antennas` file

File: src/components/HotspotPairingList.tsx [hunk 2]
```diff
@@ -25,6 +25,9 @@ const imagesByHotspotName: Record<string, any> = {
   'SyncroB.it Hotspot': (
     <Image source={require('../assets/images/syncrobit-hotspot.png')} />
   ),
+  'Finestra Miner': (
```
Comment (tyler-whitman) [inline src/components/HotspotPairingList.tsx:28]: Finestra should be below LongAP please. We keep them in the order they were added.

File: src/components/HotspotConfigurationPicker.tsx [hunk 3]
```diff
@@ -15,6 +15,37 @@ import { decimalSeparator, groupSeparator, locale } from '../utils/i18n'
...
+  | 'rak_hotspot_us'
+  | 'rak_hotspot_eu'
+  | 'nebra_outdoor'
+  | 'nebra_indoor'
+  | 'bobcat'
+  | 'syncrobit_us'
+  | 'syncrobit_eu'
+  | 'rak_custom'
+  | 'longapone_eu'
+  | 'finestra_us'
```
Comment (tyler-whitman) [inline src/components/HotspotConfigurationPicker.tsx:30]: You need to move this to https://github.com/helium/hotspot-app/blob/main/src/constants/antennas.ts#L13

File: src/components/HotspotConfigurationPicker.tsx [hunk 4]
```diff
@@ -15,6 +15,37 @@ import { decimalSeparator, groupSeparator, locale } from '../utils/i18n'
...
+  rak_hotspot_us: { id: 'rak_hotspot_us', gain: 2.3 },
+  rak_hotspot_eu: { id: 'rak_hotspot_eu', gain: 2.8 },
+  nebra_outdoor: { id: 'nebra_outdoor', gain: 3 },
+  nebra_indoor: { id: 'nebra_indoor', gain: 3 },
+  bobcat: { id: 'bobcat', gain: 4 },
+  syncrobit_us: { id: 'syncrobit_us', gain: 1.2 },
+  syncrobit_eu: { id: 'syncrobit_eu', gain: 2.3 },
+  rak_custom: { id: 'rak_custom', gain: 5.8 },
+  longapone_eu: { id: 'longapone_eu', gain: 3 },
+  finestra_us: { id: 'finestra_us', gain: 1.2 },
```
Comment (tyler-whitman) [inline src/components/HotspotConfigurationPicker.tsx:45]: You need to move this to https://github.com/helium/hotspot-app/blob/main/src/constants/antennas.ts#L30

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

File: src/features/hotspots/setup/HotspotSelectionListItem.tsx [hunk 5]
```diff
@@ -56,6 +57,8 @@ const HotspotSelectionListItem = ({
         return <BOBCAT color={svgColor} {...svgProps} />
       case 'SYNCROBIT':
         return <SYNCROBIT color={svgColor} {...svgProps} />
+      case 'Finestra':
```
Comment (tyler-whitman) [inline src/features/hotspots/setup/HotspotSelectionListItem.tsx:60]: I must have missed this one, sorry one last change this needs to be below `LONGAPONE`

File: src/features/hotspots/setup/HotspotSetupDiagnosticsScreen.tsx [hunk 6]
```diff
@@ -36,6 +36,8 @@ const HotspotSetupDiagnosticsScreen = () => {
         return 'hotspot_setup.diagnostics.bobcat_p_1'
       case 'SYNCROBIT':
         return 'hotspot_setup.diagnostics.syncrobit_p_1'
+      case 'Finestra':
```
Comment (tyler-whitman) [inline src/features/hotspots/setup/HotspotSetupDiagnosticsScreen.tsx:39]: Same with this one

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

**Comment by tyler-whitman** [comment]
I made some fixes so I opened it as another PR. https://github.com/helium/hotspot-app/pull/619

**Comment by tyler-whitman** [comment]
I made some fixes so I opened it as another PR. https://github.com/helium/hotspot-app/pull/619

**Pull request closed without merging**